doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -235,6 +235,10 @@ Error list (HTTP response code in paranthesis):
   in the request. This typically the case of requesting an entity with not enough information and the
   solution is to enhance entity identification adding more information, e.g. adding entity type
   and/or service path.
+* LengthRequired (411).  Zero/No Content-Length in PUT/POST request
+* RequestEntityTooLarge (413). Payload too large, max size supported: 1048576
```

Comment:
Note that the fiware-ngsiv2-reference.apib described NGSIv2 from an implementation agnostic point of view. In this sense, I think that 413 should be described as the error message to use in the case of large HTTP request, but the specific limit of 1048576 only applies to Orion (other NGSIv2 implementations could use a different limit).

Thus, I'm back to my original wording suggestion:

```
RequestEntityTooLarge (413). HTTP request is too large.
```